deck/templates/proposal/proposal_confirm_delete.html

```diff
@@ -0,0 +1,35 @@
+{% extends "layout.html" %}
+
+{% load i18n %}
+{% load bootstrap3 %}
+
+{% block head_title %}{% trans "Delete Proposal" %}: "{{ object }}"{% endblock %}
+
+{% block head %}
+  <!-- libraries for this page -->
+  <link href="{{ STATIC_URL }}css/libs/uniform.default.css" type="text/css" rel="stylesheet" />
+{% endblock head %}
+
+{% block content %}
+  <div class="content-wrap col-md-offset-1 col-md-10">
+    <div class="col-md-offset-3 col-md-6">
+        <h1>{% trans "Delete Proposal" %}: "{{ object }}"</h1>
+        <p>Are you sure you want to delete this proposal?</p>
```

Comment:
You need to put all text inside of the `{% trans %}` tag, because they need to be translated by the Django:
```
<p>{% trans "Are you sure you want to delete this proposal?" %}</p>
<p>{% trans "After deleting this proposal all votes will be lost forever." %}</p>
```